src/test/scala/com/atomist/rug/runtime/js/JavaScriptResponseHandlerTest.scala

```diff
@@ -41,6 +41,29 @@ class JavaScriptResponseHandlerTest extends FlatSpec with Matchers{
       |
     """.stripMargin)
 
+  val simpleResponseHandlerWithJsonCoercion =  StringFileArtifact(atomistConfig.handlersRoot + "/Handler.ts",
+    s"""
+       |import {Respond, Instruction, Response, Plan, Message} from '@atomist/rug/operations/Handlers'
+       |import {TreeNode, Match, PathExpression} from '@atomist/rug/tree/PathExpression'
+       |import {EventHandler, ParseJson, ResponseHandler, CommandHandler, Parameter, Tags, Intent} from '@atomist/rug/operations/Decorators'
+       |import {Project} from '@atomist/rug/model/Core'
+       |import {HandleResponse, HandleEvent, HandleCommand} from '@atomist/rug/operations/Handlers'
+       |
+       |@ResponseHandler("$kitties", "$kittyDesc")
+       |class KittiesResponder implements HandleResponse<any>{
+       | handle(@ParseJson response: Response<any>) : Plan {
+       |    let results = response.body() as any;
+       |
+       |    if(results.json != "is better than yaml") { throw new Error("Rats: " + results.json)}
```

Comment:
I got a little confused by the `.json` field. It's just that your payload has actually a key called `json` :)